### ISSUE TITLE
Template not found for Random forest 

### ISSUE THREAD

**Issue by devdattakhoche** [eloquentarduino/micromlgen#4]
I was trying to export random forest classifier and trying to print the code.
But it is not able to find the template , which I saw is there in the folder "randomforest/randomforest.jinja"

the code I have written 

  

    from sklearn.svm import SVC
    from micromlgen import port
    from glob import glob
    from os.path import basename
    import numpy as np
    from sklearn.ensemble import RandomForestClassifier
    
    def load_features(folder):
        dataset = None
        classmap = {}
        for class_idx, filename in enumerate(glob('%s/*.csv' % folder)):
            class_name = basename(filename)[:-4]
            classmap[class_idx] = class_name
            samples = np.loadtxt(filename, dtype=float, delimiter=',')
            labels = np.ones((len(samples), 1)) * class_idx
            samples = np.hstack((samples, labels))
            dataset = samples if dataset is None else np.vstack((dataset, samples))
    
        return dataset, classmap
    # put your samples in the dataset folder
    # one class per file
    # one feature vector per line, in CSV format
    features, classmap = load_features('data/')
    X, y = features[:, :-1], features[:, -1]
    classifier = RandomForestClassifier(random_state=0,max_depth=20).fit(X, y)
    c_code = port(classifier, classmap=classmap)
    print(c_code)

Given output : 
![Template not found](https://user-images.githubusercontent.com/49261633/102446711-6cb27f80-4054-11eb-8c5b-f19c46ce4f8f.png)

Expected Output : 

should have printed c code

**Comment by eloquentarduino** [comment]
Sorry for being late, I'm not getting email notifications from Github. Please try to install the latest version, I pushed a new one a couple days ago.

**Comment by Azkarehman** [comment]
hello, I installed the latest version and still getting the same error with random forests.

**Comment by eloquentarduino** [comment]
As a quickfix, please download the zip from Github and extract it into your project.

**Comment by eloquentarduino** [comment]
Version 1.1.22 should fix the issue.

**Comment by devdattakhoche** [comment]
Hey @eloquentarduino , If this is done , you can close this issue
